### PR TITLE
BaSD-MS-020: create router logic

### PR DESCRIPTION
**Pull request by leimeter-joaquin**
-created router logic for all entities
-admins is working as an example
-cleaned up index.js

### CODE REVIEW

File: data/admins.json [hunk 1]
```diff
@@ -1,302 +1 @@
...
...l.com","password":"ajm0IUtb"},{"id":38,"name":"Susy Mankor","email":"smankor11@e-recht24.de","password":"SAUllV4"},{"id":39,"name":"Osmund Duddin","email":"oduddin12@google.com.au","password":"XOSOfPVfeK"},{"id":40,"name":"Berton Eldredge","email":"beldredge13@example.com","password":"AZ1o4OeSS30u"},{"id":41,"name":"Dolores Crichten","email":"dcrichten14@cnbc.com","password":"rwToNHty7Mf"},{"id":42,"name":"Valida Beldum","email":"vbeldum15@phoca.cz","password":"M30JEfbWw2"},{"id":43,"name":"Alfred McEttigen","email":"amcettigen16@diigo.com","password":"Z4svfip"},{"id":44,"name":"Zonda Lacroux","email":"zlacroux17@ow.ly","password":"LVJVAz"},{"id":45,"name":"Jeramey Smithend","email":"jsmithend18@tuttocitta.it","password":"kO8ln8iNXp"},{"id":46,"name":"Regen Leyshon","email":"rleyshon19@yahoo.co.jp","password":"9KhS5lE1itY"},{"id":47,"name":"Skyler Schimon","email":"sschimon1a@live.com","password":"XGNENzIkf"},{"id":48,"name":"Anna-diane Huxster","email":"ahuxster1b@google.de","password":"dgxdtKepk6b"},{"id":49,"name":"Elsbeth Tupie","email":"etupie1c@privacy.gov.au","password":"pZQyiE3h"},{"id":50,"name":"Basilio Arrigucci","email":"barrigucci1d@columbia.edu","password":"ORdhMIb"}]
```
Comment (estefrare) [inline data/admins.json]: you shouldn't have modified the JSON

**Comment by leimeter-joaquin** [comment]
admins.json is changed because I was checking if the routes were working

File: router/applications.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+const router = express.Router()
+
+const {
+  createApplication,
+  deleteApplication,
+  listApplication,
+} = controller
+
+router.post('/', createApplication)
+router.delete('/', deleteApplication)
```
Comment (estefrare) [inline router/applications.js:13]: you have to pass the id through the path.
```router.delete('/:id', deleteApplication)```

File: router/clients.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+const {
+  createClient,
+  updateClient,
+  deleteClient,
+  listClients
+} = controller
+
+router.post('/', createClient)
+router.put('/', updateClient)
+router.delete('/', deleteClient)
```
Comment (estefrare) [inline router/clients.js:15]: you have to pass the id through the path in `updateClient` and `deleteClient` endpoints

File: router/profiles.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+const {
+  createProfile,
+  updateProfile,
+  deleteProfile,
+  listProfiles
+} = controller
+
+router.post('/', createProfile)
+router.put('/', updateProfile)
+router.delete('/', deleteProfile)
```
Comment (estefrare) [inline router/profiles.js:15]: you have to pass the id through the path in `updateProfile` and `deleteProfile` endpoints

File: router/psychologists.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+const {
+  getPsychologists,
+  createPsychologist,
+  deletePsychologist,
+  updatePsychologist
+} = controller
+
+router.post('/', createPsychologist)
+router.put('/', updatePsychologist)
+router.delete('/', deletePsychologist)
```
Comment (estefrare) [inline router/psychologists.js:15]: you have to pass the id through the path in `deletePsychologist` and `updatePsychologist` endpoints

**Review by estefrare — CHANGES_REQUESTED** [review]

**Review by estefrare — APPROVED** [review]

**Review by ignacioaristo — APPROVED** [review]

**Pull request merged**